File: Models/Models/SubCategory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Rado.Models
{
    public class SubCategory
    {
        public int subCategoryId { get; set; }
        public int categoryId { get; set; }
        public string subCategoryName { get; set; }
        public int count { get; set; }
    }
}

File: Rado/part365/src/app/model/category-subcategory/subCategory.ts
import { CONSTANT } from "@app/constant/globalLabels";

export class SubCategory {
    subCategoryId? : number;
    categoryId? : number;
    subCategoryName? : string;
    count?: number;
}

// eslint-disable-next-line @typescript-eslint/no-unused-vars
function getAll() : SubCategory {
    return {subCategoryId:0, categoryId: 0, count: 0, subCategoryName: CONSTANT.ALL};
}
